lib/convert-to-typescript-component/index.tsx

```diff
@@ -8,25 +8,25 @@ import { soupTypescriptComponentTemplate } from "./soup-typescript-component-tem
 import { convertEasyEdaJsonToTscircuitSoupJson } from "lib/convert-easyeda-json-to-tscircuit-soup-json"
 import { normalizeManufacturerPartNumber } from "lib/utils/normalize-manufacturer-part-number"
 
-export const convertRawEasyEdaToTs = (rawEasy: any) => {
+export const convertRawEasyEdaToTs = async (rawEasy: any) => {
```

Comment:
Is `convertRawEasyEdaToTs ` heavily used across the codebase? I'm worried that now this function would need to be awaited elsewhere where it's already used (after version update).

Reply:
@seveibar wdyt?

Reply:
i don't think so!

Reply:
fwiw the way i see it- if we break other codebases with automatic version upgrades that means that the repo didn't have good enough tests on CI- since the CI should not release if tests/build is failing. Discovering which repos lack testing is a good outcome even if it breaks things